metrics/metrics.yaml

```
@@ -1,5 +1,15 @@
-- metric-name: "buffered_read/download_block_latency"
-  description: "The cumulative distribution of block download latencies, along with status: successful, cancelled, or failed."
+- metric-name: "buffered_read/fallback_trigger_count"
+  description: "The cumulative number of times the BufferedReader falls back to a different reader, along with the reason: random_read_detected or insufficient_memory."
+  type: "int_counter"
+  attributes: &buffered_read_fallback_attributes
```

Not required unless you want to reuse the list later on.
```suggestion
  attributes:
```